eu.nokun.MirrorHall.json

```diff
@@ -0,0 +1,104 @@
+{
+    "app-id": "eu.nokun.MirrorHall",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "46",
+    "add-extensions": {
+        "org.freedesktop.Platform.ffmpeg-full": {
+            "version": "23.08",
+            "directory": "lib/ffmpeg",
+            "add-ld-path": ".",
+            "no-autodownload": false,
+            "autodelete": false
+        },
+        "org.freedesktop.Platform.GStreamer.gstreamer-vaapi": {
+            "directory": "lib/gstreamer-1.0",
+            "version": "23.08",
+            "autodelete": false,
+            "no-autodownload": true,
+            "add-ld-path": "lib",
+            "download-if": "have-intel-gpu",
+            "autoprune-unless": "have-intel-gpu"
+        }
+
+    },
+    "sdk": "org.gnome.Sdk",
+    "command": "mirrorhall",
+    "finish-args": [
+        "--share=network",
+        "--share=ipc",
+        "--device=dri",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--filesystem=xdg-run/pipewire-0",
+        "--talk-name=org.gnome.Mutter.ScreenCast",
+        "--env=GST_DEBUG=2",
+        "--env=GST_PLUGIN_PATH=/app/lib/gstreamer-1.0:/usr/lib/gstreamer-1.0"
+    ],
+    "cleanup": [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "cleanup-commands": [
+        "mkdir -p ${FLATPAK_DEST}/lib/ffmpeg"
+    ],
+    "modules": [
+        "python3-requirements.json",
+        {
+            "name": "inetutils",
+            "buildsystem": "simple",
+            "build-commands": [
+                "./configure --prefix=/app --libexec=/app/bin --localstatedir=/var --sysconfdir=/etc --without-{wrap,pam} --disable-{dnsdomainname,ftp,ftpd,ifconfig,inetd,logger,ping,ping6} --disable-{rcp,rexec,rexecd,rlogin,rlogind,rsh,rshd,syslogd} --disable-{talk,talkd,telnet,telnetd,tftp,tftpd,traceroute,uucpd,whois} --enable-hostname",
+                "make -j ${FLATPAK_BUILDER_N_JOBS}",
+                "install -Dm755 ./src/hostname /app/bin/hostname"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://ftp.gnu.org/gnu/inetutils/inetutils-2.0.tar.xz",
+                    "sha256": "e573d566e55393940099862e7f8994164a0ed12f5a86c3345380842bdc124722"
+                }
+            ]
+        },
+        {
+	    "name": "clapper",
+	    "buildsystem": "meson",
+	    "sources": [
+	        {
+		    "type": "git",
+		    "url": "https://github.com/Rafostar/clapper",
+		    "tag": "0.6.1"
```

Comment:
why are you bundling another player?

Reply:
This is to use clappersink, the pipeline player widget. This will not be required anymore after GNOME 47 is released.